kubernetes/cmsweb/scripts/encrypt-secrets.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# helper script to encrypt services secrets file using sops-age keys mounted as secrets in Kubernetes clusters
+if [ $# -ne 2 ]; then
+  echo "The required parameters are missing. Please use encrypt-secrets.sh <namespace> <full-path-to-services-secret-file>"
+  exit 1;
+fi
+
+namespace=$1
+secret_file=$2
+echo "Namespace: $namespace"
+echo "File to be encrypted: $secret_file"
+
+tmpDir=/tmp/$USER/sops
+if [ -d $tmpDir ]; then
+  rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+
+### Download sops binary if it does not exist in the system
+if [ -z "`command -v sops`" ]; then
+  # download soap in tmp area
+  wget -O sops https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.linux.amd64
```

Comment:
Since we download new script in user's home/bin area I suggest to put appropriate message here, e.g.
`echo "Download and install sops scripts under $HOME/bin"`